Turkey/TestRunner.cs

```diff
@@ -161,5 +162,22 @@ public async Task<TestResults> ScanAndRunAsync(List<TestOutput> outputs, string
 
             return results;
         }
+
+        private static void LogEnvironment(string logDir)
+        {
+            foreach (var sourcePath in new[] { "/proc/cpuinfo", "/proc/meminfo", "/etc/os-release" })
+            {
+                if (File.Exists(sourcePath))
+                {
+                    string destinationPath = Path.Combine(logDir, Path.GetFileName(sourcePath));
+                    // Note: we don't use File.Copy because that copies the file permissions,
+                    //       which gives issues on successive runs since the files are non-writable.
+                    File.WriteAllBytes(destinationPath, File.ReadAllBytes(sourcePath));
+                }
+            }
+
+            string unameOutput = ProcessRunner.Run("uname", "-a");
+            File.WriteAllText(Path.Combine(logDir, "uname"), unameOutput);
+        }
```

Comment:
Do you think it makes sense to augment this with the versions of packages installed via `rpm -qa` or some such?